tools/utils/template_locals.ts

```diff
@@ -1,8 +1,9 @@
-import {APP_BASE, APP_DEST, APP_ROOT, APP_TITLE, SYSTEM_CONFIG, VERSION, HOT_LOADER_PORT, BOOTSTRAP_MODULE} from '../config';
+import {APP_BASE, APP_DEST, APP_ROOT, APP_TITLE, SYSTEM_CONFIG, VERSION, HOT_LOADER_PORT, BOOTSTRAP_MODULE, ENV} from '../config';
```

Comment:
Don't you think we should `import * as CONFIG from ...` and return `CONFIG` so we never have to touch this ?